src/Lean/Parser/Term.lean

```diff
@@ -568,7 +568,7 @@ Being borrowed only affects the ABI and runtime behavior of the function when co
 
 When a function argument is borrowed, the function does not consume the value. This means that the function will not decrement the value's reference count or deallocate it, and the caller is responsible for doing so.
 
-Please see https://lean-lang.org/lean4/doc/dev/ffi.html#borrowing for a complete description.
+Please see https://github.com/leanprover/lean4/blob/master/doc/dev/ffi.md#borrowing for a complete description.
```

Comment:
Maybe instead
https://lean-lang.org/doc/reference/latest/Run-Time-Code/Foreign-Function-Interface/#ffi-borrowing
?

Reply:
I agree that we should delete duplicated docs, after reviewing to make sure they agree with each other :-)

Make sure to link to the permalink: https://lean-lang.org/doc/reference/latest/find/?domain=Verso.Genre.Manual.section&name=ffi-borrowing